### PR TITLE
BIP-592: Exit and Withdraw from BAL AURA ARB Pool Arbitrum

### PR DESCRIPTION
**Pull request by Zen-Maxi**
Instructions: https://docs.google.com/document/d/14Yaey2GbFvxN0ajrW9Tao7RlNXm6UZrN99ON7ejf4IA/edit

Sim: https://dashboard.tenderly.co/public/safe/safe-apps/simulator/dc2a26ad-f60f-42f6-aa6a-8b2561eb68c7

#### Specification:

The DAO Multisig on Arbitrum `0xaF23DC5983230E9eEAf93280e312e57539D098D0` will interact with the AuraArbBalGrant contract at `0x8D803f7f7e26E586ee90E5A872cf7830e21f7727` and write the `exit` function and for `_minOuts` will pass the return amounts in uint256 format from the queryExit of the poolId `0xbcaa6c053cab3dd73a2e898d89a4f84a180ae1ca000100000000000000000458` for a proportional exit for BAL, AURA, and ARB respectively.

Following the exit, in the same transaction,, the DAO multisig `0xaF23DC5983230E9eEAf93280e312e57539D098D0` will also write the same contract, `0x8D803f7f7e26E586ee90E5A872cf7830e21f7727` function `withdrawBalances` to receive the tokens to the multisig.

For Reference:

AuraArbBalGrant Contract https://arbiscan.io/address/0x8d803f7f7e26e586ee90e5a872cf7830e21f7727#readContract

BPT Held: https://arbiscan.io/token/0xbcaa6c053cab3dd73a2e898d89a4f84a180ae1ca?a=0x8d803f7f7e26e586ee90e5a872cf7830e21f7727

`balanceOf` returns `3242539693298704732855523`, 3,242,539.693 Pool tokens.

At the time of writing `minOuts` would be:

[`407389945800000000000000`, `2376519156700000000000000`, `1361586315300000000000000`]

https://dashboard.tenderly.co/public/safe/safe-apps/simulator/dc2a26ad-f60f-42f6-aa6a-8b2561eb68c7

This is subject to change at the time of payload execution.

### CODE REVIEW

File: BIPs/2024-W18/BIP-592 Exit and Withdraw from BAL/AURA/BIP-592.json [hunk 1]
```diff
@@ -29,6 +29,17 @@
       "contractInputsValues": {
         "_minOuts": "[396663000000000000000000, 2299087000000000000000000, 1360655000000000000000000]"
       }
+    },
+    {
+      "to": "0x8D803f7f7e26E586ee90E5A872cf7830e21f7727",
```
Comment (Xeonus) [inline BIPs/2024-W18/BIP-592 Exit and Withdraw from BAL/AURA/BIP-592.json:34]: Checks out with https://github.com/BalancerMaxis/bal_addresses/blob/4396ca6ee46272111bb6db65e5054af079e58dda/extras/arbitrum.json#L64

**Review by Xeonus — DISMISSED** [review]
Call method looks good, verified on contract

**Review by Tritium-VLK — DISMISSED** [review]
This should be executed with private RPC>

**Comment by Tritium-VLK** [comment]
Concerned that 2% is not enough room, prices could move, and this tx could start to revert.  This would cuase the whole batch to have to be replaced.

1: Consider wider ranges
2: if theya re not really wide run it's own tx/do not batch.

**Comment by gosuto-inzasheru** [comment]
this was discussed already right? numbers need to be adjusted as close to the actual posting of the tx as possible. and yea that could be done with 5% slippage or whatever as well

**Comment by Tritium-VLK** [comment]
Would be nice if this were in the BIP, specific details.  I don't like making unilateral with the DAOs resources.  My job here is only to execute what is instructed by governance, not to "think or interpret it".  

In this case I'll get a few approvals by Maxis on the final TX as I am not comfortable taking this decision without boundaries set/approved by governance.

**Comment by Tritium-VLK** [comment]
I suggest 10% below market price and we exec with private RPC.  There's some risk someone could exec before us after the last siggy, but it seems very low risk.

Current [tenderly](https://dashboard.tenderly.co/public/safe/safe-apps/simulator/98a774f9-ef1b-4057-95c7-5f7d6eeb5208)
90% would be:
`[36283773870000002039808, 2058925925750633987571712, 128719428660000012107776']

File: BIPs/2024-W18/BIP-592 Exit and Withdraw from BAL/AURA/BIP-592.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            "internalType": "uint256[]",
+            "name": "_minOuts",
+            "type": "uint256[]"
+          }
+        ],
+        "name": "exit",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "_minOuts": "[396663000000000000000000, 2299087000000000000000000, 1360655000000000000000000]"
```
Comment (Tritium-VLK) [inline BIPs/2024-W18/BIP-592 Exit and Withdraw from BAL/AURA/BIP-592.json:30]:
```suggestion
        "_minOuts": "[36283773870000002039808, 2058925925750633987571712, 128719428660000012107776]"
```

Comment (Tritium-VLK) [reply BIPs/2024-W18/BIP-592 Exit and Withdraw from BAL/AURA/BIP-592.json:30]: Changing as per comment.

Comment (Tritium-VLK) [reply BIPs/2024-W18/BIP-592 Exit and Withdraw from BAL/AURA/BIP-592.json]: Commited into branch/new report should have final minouts.

**Comment by Tritium-VLK** [comment]
See prettified report here, pending approval of logic change PR:

https://github.com/BalancerMaxis/multisig-ops/blob/10500d16d508a5f93c6d47a62dd5da37be307d90/BIPs/4269-W69/BIP-592.report.txt

File: BIPs/2024-W18/BIP-592 Exit and Withdraw from BAL/AURA/BIP-592.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            "internalType": "uint256[]",
+            "name": "_minOuts",
+            "type": "uint256[]"
+          }
+        ],
+        "name": "exit",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "_minOuts": "[36283773870000002039808, 2058925925750633987571712, 128719428660000012107776]"
```
[the remark this answers is not in the document]
Comment (Tritium-VLK) [reply BIPs/2024-W18/BIP-592 Exit and Withdraw from BAL/AURA/BIP-592.json:30]: @mikebmikeb pointed out that I mismultiplied 2 of the minouts.

**Comment by Tritium-VLK** [comment]
image

>>> aura=2274285
>>> bal=401210
>>> arb=1424835
>>> [int(0.9e18*bal), int(0.9e18*aura), int(0.9e18*arb)]
[361088999999999976669184, 2046856500000000053346304, 1282351499999999948750848]

**Review by mikebmikeb — APPROVED** [review]

**Pull request merged**